galata/test/jupyterlab/sidebars.test.ts

```diff
@@ -194,3 +201,157 @@ test.describe('Sidebars', () => {
     expect(tableOfContentsElementRole).toEqual('region');
   });
 });
+
+const runningSessionsElementAriaLabels: string[] = [
+  'Open Tabs Section',
+  'Kernels Section',
+  'Language servers Section',
+  'Terminals Section'
+];
+const debuggerElementAriaLabels: string[] = [
+  'Variables Section',
+  'Callstack Section',
+  'Breakpoints Section',
+  'Source Section',
+  'Kernel Sources Section'
+];
+const propertyElementAriaLabels: string[] = [
+  'Warning Section',
+  'Installed Section',
+  'Discover Section'
+];
+
+const sessionsAndDebuggerAriaLabels: string[] =
+  runningSessionsElementAriaLabels.concat(debuggerElementAriaLabels);
+
```

Comment:
We could use a structure like 
```ts
elementAriaLabels = {
  'jp-running-sessions': [
    'Open Tabs Section',
    '...'
  ],
  'jp-debugger-sidebar': [
    '...',
  ],
  'extensionmanager.main-view': [
    '...',
  ]
}
```
and have only one test function, looping first on tab name and then on section name.
And maybe something similar can be done for left and right sidebars.